docs/USING.rst

```diff
@@ -50,3 +50,41 @@ to hard-disk booting the installed system.
 This is, as the filepath states, merely an example of how you might deploy to
 physical nodes.  Existing deployment systems such as Canonical's MAAS have also
 been run succesfully.
+
+Using CURL Commands
+====================
```

Comment:
Remember, this isn't actually plain text, it's RST. Have a look at the rich text view of this file and you'll see that the formatting on the code samples below doesn't look right; please fix. Github lets you edit/commit files on the website, and it has a preview option, so it's useful for stuff like this.